### PR TITLE
restore: update full restore speed

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Leavrth — APPROVED** [review]

**Review by BornChanger — APPROVED** [review]

File: br/br-pitr-guide.md [hunk 1]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
...
 >
 > 以上功能指标是根据下述两个场景测试得出的结论，如有出入，建议以实际测试结果为准：
 >
-> - 全量恢复速度 = 全量恢复数据量 /（时间 * TiKV 数量）
-> - 日志恢复速度 = 日志恢复总量 /（时间 * TiKV 数量）
+> - 全量恢复速度 = 真实恢复数据量 /（时间 * TiKV 数量）
+> - 日志恢复速度 = 真实日志恢复总量 /（时间 * TiKV 数量）
 >
-> 其中全量恢复数据量，是指单个副本中所有 KV 的逻辑大小，并不代表实际恢复的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
+> 在共享存储中，只存放单个副本中所有 KV 的逻辑大小，这个数据并不代表实际恢复的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
```
Comment (BornChanger) [inline br/br-pitr-guide.md:118]:
```suggestion
> 在外部存储中，只存放单个副本中所有 KV，它并不代表集群实际恢复的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
```

File: br/br-pitr-guide.md [hunk 2]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
...
 - 使用 `tiup br log truncate` 清理过期的日志备份数据速度为 600 GB/h
 
 > **注意：**
 >
 > 以上功能指标是根据下述两个场景测试得出的结论，如有出入，建议以实际测试结果为准：
 >
-> - 全量恢复速度 = 全量恢复数据量 /（时间 * TiKV 数量）
-> - 日志恢复速度 = 日志恢复总量 /（时间 * TiKV 数量）
+> - 全量恢复速度 = 真实恢复数据量 /（时间 * TiKV 数量）
+> - 日志恢复速度 = 真实日志恢复总量 /（时间 * TiKV 数量）
```
Comment (BornChanger) [inline br/br-pitr-guide.md:116]:
```suggestion
> - 全量恢复速度 = 集群上全部 tikv 节点恢复数据量 /（时间 * TiKV 数量）
> - 日志恢复速度 = 集群上全部 tkv 节点日志恢复总量 /（时间 * TiKV 数量）
```

File: br/backup-and-restore-overview.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
 
 #### 恢复的性能
 
-- 恢复集群快照数据备份，速度可以达到单 TiKV 存储节点 100 MiB/s，恢复速度具有可扩展性。更详细说明请参考[恢复性能和影响](/br/br-snapshot-guide.md#快照恢复的性能与影响)。
+- 恢复集群快照数据备份，速度可以达到单 TiKV 存储节点 1 GB/s，恢复速度具有可扩展性。更详细说明请参考[恢复性能和影响](/br/br-snapshot-guide.md#快照恢复的性能与影响)。
```
Comment (lilin90) [inline br/backup-and-restore-overview.md:97]:
```suggestion
- 恢复集群快照数据备份，速度可以达到单 TiKV 存储节点 1 GiB/s，恢复速度具有可扩展性。更详细说明请参考[恢复性能和影响](/br/br-snapshot-guide.md#快照恢复的性能与影响)。
```

File: br/br-pitr-guide.md [hunk 4]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
 
 ## PITR 的性能指标
 
-- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 280 GB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GB/h
+- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 2 TB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GB/h
```
Comment (lilin90) [inline br/br-pitr-guide.md:108]:
```suggestion
- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 2 TiB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GB/h
```

File: br/br-pitr-guide.md [hunk 5]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
...
 - 使用 `tiup br log truncate` 清理过期的日志备份数据速度为 600 GB/h
 
 > **注意：**
 >
 > 以上功能指标是根据下述两个场景测试得出的结论，如有出入，建议以实际测试结果为准：
 >
-> - 全量恢复速度 = 全量恢复数据量 /（时间 * TiKV 数量）
-> - 日志恢复速度 = 日志恢复总量 /（时间 * TiKV 数量）
+> - 全量恢复速度 = 集群上全部 tikv 节点恢复数据量 /（时间 * TiKV 数量）
+> - 日志恢复速度 = 集群上全部 tkv 节点日志恢复总量 /（时间 * TiKV 数量）
```
Comment (lilin90) [inline br/br-pitr-guide.md:116]: tkv 是想写 TiKV 吗？
```suggestion
> - 全量恢复速度 = 集群上全部 TiKV 节点恢复数据量 /（时间 * TiKV 数量）
> - 日志恢复速度 = 集群上全部 TiKV 节点日志恢复总量 /（时间 * TiKV 数量）
```

File: br/backup-and-restore-overview.md (hunk 3 above)
Comment (qiancai) [inline br/backup-and-restore-overview.md:97]:
```suggestion
- 恢复集群快照数据备份，速度可以达到单 TiKV 存储节点 1 GiB/s，恢复速度具有可扩展性。更详细说明请参考[恢复性能和影响](/br/br-snapshot-guide.md#快照恢复的性能与影响)。
```

File: br/br-pitr-guide.md (hunk 4 above)
Comment (qiancai) [inline br/br-pitr-guide.md:108]:
```suggestion
- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 2 TiB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GiB/h
```

File: br/br-pitr-guide.md (hunk 5 above)
Comment (qiancai) [inline br/br-pitr-guide.md:116]:
```suggestion
> - 全量恢复速度 = 集群中所有 TiKV 节点恢复数据总量 /（时间 * TiKV 数量）
> - 日志恢复速度 = 集群上所有 TiKV 节点日志恢复总量 /（时间 * TiKV 数量）
```

File: br/br-pitr-guide.md [hunk 6]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
...
 >
-> 其中全量恢复数据量，是指单个副本中所有 KV 的逻辑大小，并不代表实际恢复的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
+> 在外部存储中，只存放单个副本中的 KV 数据，它并不代表集群实际恢复后的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
```
Comment (qiancai) [inline br/br-pitr-guide.md:118]:
```suggestion
> 外部存储中只存放单个副本中的 KV 数据，因此外部存储上的数据量并不代表集群实际恢复后的数据量。BR 恢复数据时会根据集群设置的副本数来恢复全部副本，当副本数越多时，实际恢复的数据量也就越多。
```

File: br/br-pitr-guide.md [hunk 7]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
 
 ## PITR 的性能指标
 
-- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 280 GB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GB/h
+- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 2 TiB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GB/h
```
Comment (qiancai) [inline br/br-pitr-guide.md:108]:
```suggestion
- PITR 恢复速度，平均到单台 TiKV 节点：全量恢复 (Full Restore) 为 2 TiB/h，日志恢复（Restore Meta Files 和 Restore KV Files）为 30 GiB/h
```

File: br/br-pitr-guide.md [hunk 8]
```diff
@@ -105,17 +105,17 @@ Restore KV Files <--------------------------------------------------------------
...
 - 使用 `tiup br log truncate` 清理过期的日志备份数据速度为 600 GB/h
 
 > **注意：**
 >
 > 以上功能指标是根据下述两个场景测试得出的结论，如有出入，建议以实际测试结果为准：
 >
-> - 全量恢复速度 = 全量恢复数据量 /（时间 * TiKV 数量）
-> - 日志恢复速度 = 日志恢复总量 /（时间 * TiKV 数量）
+> - 全量恢复速度 = 集群中所有 TiKV 节点恢复数据总量 /（时间 * TiKV 数量）
+> - 日志恢复速度 = 集群上所有 TiKV 节点日志恢复总量 /（时间 * TiKV 数量）
```
Comment (qiancai) [inline br/br-pitr-guide.md:116]:
```suggestion
> - 日志恢复速度 = 集群中所有 TiKV 节点日志恢复总量 /（时间 * TiKV 数量）
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by BornChanger — APPROVED** [review]

**Pull request merged**